script/lib/git.py

```diff
@@ -40,7 +40,7 @@ def get_repo_root(path):
   return get_repo_root(parent_path)
 
 
-def apply(repo, patch_path, directory=None, index=False, reverse=False):
+def apply_patch(repo, patch_path, directory=None, index=False, reverse=False):
```

Comment:
Why the rename?

Reply:
because otherwise, pylint.py warns that this is overridding the `apply` builtin

Reply:
Oh, ok =/